src/com/xilinx/rapidwright/design/DRC.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (c) 2021 Xilinx, Inc.
+ * All rights reserved.
+ *
+ * Author: Eddie Hung, Xilinx Research Labs.
+ *
+ * This file is part of RapidWright.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.xilinx.rapidwright.design;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.xilinx.rapidwright.design.drc.DesignRuleCheckInterface;
+import com.xilinx.rapidwright.design.drc.NetRoutesThruLutAtMostOnce;
+
+public class DRC implements DesignRuleCheckInterface {
+    public static final List<Class<? extends DesignRuleCheckInterface>> checks =
+            new ArrayList<Class<? extends DesignRuleCheckInterface>>() {{
+        add(NetRoutesThruLutAtMostOnce.class);
+    }};
+
+    @Override
+    public int run(Design design) {
+        // DesignRuleCheckInterface::run() returns an int of how many checks failed,
+        // sum those up
+        return checks.stream().map((c) -> {
+                    try {
+                        DesignRuleCheckInterface i = c.getDeclaredConstructor().newInstance();
+                        return (int) c.getDeclaredMethod("run", Design.class).invoke(i, design);
+                    } catch (Exception e) {
+                        throw new RuntimeException(e);
+                    }
+                })
+                .reduce(0, Integer::sum);
+    }
+
+    private static void printUsageAndExit() {
+        System.out.println("USAGE: <input.dcp> [--strict]");
+        System.exit(1);
```

Comment:
The `Security Manager` is being [deprecated in Java 17](https://openjdk.java.net/jeps/411) and will be eventually removed.  There is a possibility of [a new API that could catch `System.exit()` calls](https://bugs.openjdk.java.net/browse/JDK-8199704), but no activity on it for a while.  If we do go down this path, it will dispense warning messages and will get progressively harder to use in later versions of Java.

Reply:
This is an example of the warning message users would see each time they `import rapidwright`:
```
WARNING: A terminally deprecated method in java.lang.System has been called
WARNING: System::setSecurityManager has been called by com.xilinx.rapidwright.util.PythonSecurityManager (file:/C:/home/workspace/RapidWright/bin/)
WARNING: Please consider reporting this to the maintainers of com.xilinx.rapidwright.util.PythonSecurityManager
WARNING: System::setSecurityManager will be removed in a future release
```

Reply:
Moved conversation to #283